ganga/GangaDirac/Lib/Backends/DiracBase.py

```diff
@@ -1311,6 +1317,10 @@ def finalise_jobs(allJobs, downloadSandbox=True):
         jobs = [stripProxy(j) for j in theseJobs]
         manager = AsyncDiracManager()
         for job in jobs:
+            print(f'Attempting to add from DIRAC task for {finished_job}')
+            if not monitoring_component.enabled:
+                print('NOPERZ')
```

Comment:
again raw prints rather than logging messages.

Reply:
Yeah I left them in by accident when testing out, removed now, thanks for noticing!